.github/workflows/gradle.yml

```diff
@@ -24,10 +24,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up JDK 11
+    - name: Set up JDK
       uses: actions/setup-java@v3
       with:
-        java-version: '11'
+        java-version: '18'
```

Comment:
Seems a curious choice... why not 17 or 21 for LTS?

Reply:
Ah, I thought 18 was the LTS version, apologies